src/api/store/community.py

```diff
@@ -193,19 +198,48 @@ def create_community(self,context: Context, args) -> (dict, MassEnergizeAPIError
         new_action.is_published = False
         new_action.is_global = False
 
-        #make sure the action title does not exceed the max length expected
-        suffix = f'-{new_community.subdomain}-{action_to_copy_id}'
-        new_action.title = new_action.title[:SHORT_STR_LEN-len(suffix)]+suffix 
+        # remove the "TEMPLATE", "TEMP" or "TMP" prefix or suffix
+        old_title = new_action.title
```

Comment:
what is the goal of this?

Reply:
The goal of this is to have the title of the new action be simple, just the name of the template action with "-Concord" on the end.  Makes the cadmin's job easier and there is no reason to have "TEMPLATE" as part of the name or "Copy XXXX".  The comment explains that.